JsonSchema/RecursiveRefKeyword.cs

```diff
@@ -55,11 +55,15 @@ public void Evaluate(EvaluationContext context)
 			if (scopeRoot == null)
 				throw new Exception("This shouldn't happen");
 
-			if (scopeRoot.RecursiveAnchor == null) continue;
+			if (scopeRoot is not JsonSchema schemaRoot)
+				throw new Exception("Does OpenAPI use anchors?");
```

Comment:
Not sure this is entire relevant here since `$recursiveRef` is exclusively 2019-09.